redash/models/users.py

```diff
@@ -187,8 +187,14 @@ def get_by_api_key_and_org(cls, api_key, org):
         return cls.get_by_org(org).filter(cls.api_key == api_key).one()
 
     @classmethod
-    def all(cls, org):
-        return cls.get_by_org(org).filter(cls.disabled_at.is_(None))
+    def all(cls, org, is_invitation_pending=None):
+        condition = True  # no filter
+        if is_invitation_pending is not None:
+            if is_invitation_pending:
+                condition = cls.is_invitation_pending.is_(True)  # pending
+            else:
+                condition = cls.is_invitation_pending.isnot(True)  # not pending (check for both `false`/`null`)
+        return cls.get_by_org(org).filter(cls.disabled_at.is_(None), condition)
```

Comment:
What you can do instead is:

```python
users = cls.get_by_org(org).filter(cls.disabled_at.is_(None))

if is_invitiation_pending is not None:
    users = users.filter(cls.is_invitation_pending.is_(is_invitation_pending)) 

return users
```

You should _not_ check for `null` value, because existing users have a null value (@rauchy correct me if I'm wrong).

Reply:
Yes, yes, YES!! That's exactly what I need. Thanks!